Contrast-Security-OSS/contrast-eclipse-plugin
# TS-16457 modify the eclipse plugin publish action workflow to sign the artifacts before upload to the update site

Changes:
publish.yml 
pfx_to_file.yml

Motivation:

- the publish action which updates the plugin jars to the eclipse update site now sign the jar files to avoid the warning message when the eclipse plug-in is downloaded.
- Because this change affects directly the files downloaded by clients was created a similar action to allow the reviewes test and verify the signing

How To Test:

- Run the PFX to File action over this branch.
- When the workflow finish download the artifacts created (link at the bottom)
- Using jarsigner run on a command window: jarsigner -verify file_to_verifiy.jar on the plugins and features jars
